webapp/security/views.py

```diff
@@ -258,3 +259,22 @@ def api_create_notice():
     db_session.commit()
 
     return flask.jsonify({"message": "Notice created"}), 201
+
+
+# CVE views
+# ===
+
+
+def cve(cve_id):
+    print(cve_id)
+    match_cves = re.compile(r"cve-\d{4}-\d{4,7}")
+    if cve_id == "cve":
```

Comment:
Why not a view for the list and another view for a single one?

Reply:
Design reasons https://docs.google.com/document/d/1VuAElUpFay7MBFNmE3NhSyY9TkXiqHcA1ZHQ11AYO-Q/edit#

Reply:
Can you paste another link? I just get at the top of the document.

Reply:
https://docs.google.com/document/d/1VuAElUpFay7MBFNmE3NhSyY9TkXiqHcA1ZHQ11AYO-Q/edit#heading=h.zcaul6pvo5w2

Reply:
here it seems you could do:

``` python
# webapp/app.py
app.add_url_rule("/security/cve", view_func=cve_list)
app.add_url_rule("/security/<regex('cve-\d{4}-\d{4,7}'):cve_id>", view_func=cve)

# webapp/security/views.py
def cve_list():
  return flask.render_template("security/cve_list.html")
def cve(cve_id):
   return flask.render_template("security/cve_single.html")
```

Reply:
Neat! Done.

Reply:
Nice @tbille , I almost mentioned before to create the <regex> expression rule, I didn't know that we already have it. Is it because flask-base or it is now included in flask?